.github/workflows/images.yml

```diff
@@ -110,115 +110,3 @@ jobs:
           push: true
           no-cache: true
           tags: prairielearn/grader-r:latest,prairielearn/grader-r:${{ github.sha }}
-
```

Comment:
This entire section is deleted as I am reusing image-builder for both PRs and master.